### ISSUE TITLE
Write out a comparison page with RMarkdown

### ISSUE THREAD

**Issue by choldgraf** [ExecutableBookProject/myst_parser#77]
I've had a few people ask me if / how MyST markdown is different from RMarkdown. Particularly because they use similar patterns for configuration etc.

We should add a page that compares how to do the same thing (more or less) in both flavors of markdown, to make it easier for people to see the similarities and differences. This is also an opportunity to decide for ourselves if this is a more generic and well-structured improvement to RMarkdown :-)

I wonder if that's something that one of the integration developers could work on? what do you think @jstac ?

**Comment by mmcky** [comment]
I think this is a nice idea and will be a useful resource in comparison of markup.

**Comment by mmcky** [comment]
@choldgraf where should this live? I guess in this project docs as it technically defines `myst`.

**Comment by choldgraf** [comment]
Yeah I'd imagine in the MyST parser docs for now...we will probably need to do a little refactor/rearrange at some point anyway
